Bowling/NUnitTestBowling/BowlingTests.cs

```diff
@@ -0,0 +1,70 @@
+using NUnit.Framework;
+using System.Runtime.InteropServices;
+using BowlingApp;
+using System.IO;
+using System;
+using System.Collections.Generic;
+
+namespace NUnitTestBowling
+{
+    public class Tests
+    {
+
+        [Test]
+        [TestCase("XXXXXXXXXXXX", "300", Description = "Perfect Game")]
+        [TestCase("9-9-9-9-9-9-9-9-9-9-", "90", Description = "Open Frames with -")]
+        [TestCase("5/5/5/5/5/5/5/5/5/5/5", "150", Description = "All Spares")]
+        [TestCase("548/XX725/7/52X6/5", "158", Description = "Random Open Spares Strikes")]
+        [TestCase("54637281233445625381", "82", Description = "All Open")]
+        public void TestCompleteScoringOfInputString(string marks, string expectedScore)
+        {
+            using (var sw = new StringWriter())
+            {
+                // user string for bowling record
+                using (var sr = new StringReader(marks))
+                {
+                    Console.SetOut(sw);
+                    Console.SetIn(sr);
+
+                    Bowling bowling = new Bowling();
+                    bowling.stepsForGameCalculation();
+
+                    var result = sw.ToString();
+                    Assert.IsTrue(result.Contains(expectedScore));
+                }
+            }
+        }
+
+        [Test]
+        public void TestManualCompleteScoringOfInputString()
```

Comment:
While you've done a fairly good job adhering to [Arrange Act Assert](http://wiki.c2.com/?ArrangeActAssert) in the structuring of your test, let's try to get the names of your tests to reflect that as well. In general, our tests are named following this naming pattern: `MethodUnderTest_Preconditions_ExpectedResults`. That gives us a pretty good idea of what the test is doing, what it's assuming, what it expects, and inevitably why it failed without even reading the body of the method.